### PR TITLE
Point flower to github repo

### PR DESCRIPTION
**Pull request by AmitPhulera**
## Product Description

We fixed one of the serialization issues in the flower repo and the fix was accepted but a new version(>1.2.0) has not been released yet. So we are temporarily pointing flower to its GitHub repo.
This would be reverted when a new version of flower is released.
## Technical Summary

https://dimagi-dev.atlassian.net/browse/SAAS-13911

## Safety Assurance
A dependency upgrade to fix an existing issue with flower. 

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: requirements/prod-requirements.in [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 pip
 
-flower
+# Switch back to normal package as soon as flower realease a version > 1.2.0
```
Comment (joxl) [inline requirements/prod-requirements.in:5]: nit

```suggestion
# Switch back to normal package as soon as flower releases a version > 1.2.0 on pypi
```

File: requirements/prod-requirements.in [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+flower @ git+https://github.com/mher/flower
```
Comment (joxl) [inline requirements/prod-requirements.in:6]: I recommend pinning a specific ref. A tag would be better, but they haven't pushed one yet, so I've used the SHA of your PR's merge commit.

```suggestion
flower @ git+https://github.com/mher/flower@e06f2750
```

**Review by joxl — CHANGES_REQUESTED** [review]
Requesting changes because unpinned (non-internal) github dependencies are unsafe.

**Review by joxl — APPROVED** [review]

**Comment by calellowitz** [comment]
I think this is probably fine, but typically we would instead fork the repo and point to a commit on our fork, in the admittedly unlikely event that there is a force push or other change to history in the main repo. It is also just a little nicer having it be a repo we control.

File: requirements/prod-requirements.txt [hunk 3]
```diff
@@ -192,7 +192,7 @@ eulxml==1.1.3
     # via -r base-requirements.in
 executing==0.8.3
     # via stack-data
-flower==1.2.0
+flower @ git+https://github.com/mher/flower@a4fc7c97d897a3172d1058d5269115eb88d811de
```
Comment (dannyroberts) [inline requirements/prod-requirements.txt:195]: Here's the diff for anyone who wants to look over it. Two-way diff: https://github.com/mher/flower/compare/v1.2.0..a4fc7c97d897a3172d1058d5269115eb88d811de. One-way diff https://github.com/mher/flower/compare/v1.2.0...a4fc7c97d897a3172d1058d5269115eb88d811de.

Looks fine to me

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**